### DISCUSSION TITLE
Older Firmeware Archive

### DISCUSSION THREAD

**Discussion by ru320** [Koenkk/Z-Stack-firmware#429] [Ideas]
Hello

I have more a wish than a problem, but maybe you have me a solution.

I have a Homeassistant system running, but I am dissatisfied and want to reboot.

Currently I have 2 systems and therefore 2 CC2531 sticks.

The old stick (firmware 20190608) works fine in both systems.

The new stick (firmware 20211116) can detect and integrate Zigbee devices, but the interview fails. 
When I use the old stick I can connect the devices without any problems.

Until my homeassitant move is complete, I want to run both systems.

Hence my questions:
1. is my problem known and what can I do?
2. is there an archive of the different firmwares?
3. is it possible to get the old firmware somewhere? unfortunately I have no backup.

Greetings from Bolheim
Rü

Translated with www.DeepL.com/Translator (free version

**Comment by Koenkk** [comment]
20190608 can be found here: https://github.com/Koenkk/Z-Stack-firmware/tree/4ab0d66e28b10dba75bcc8f1f5a7d7af4fbabf6c/coordinator/Z-Stack_Home_1.2/bin/default

**Reply by ru320** [reply]
Thank you,
you are incredibly fast
I will test this right away

**Comment by ru320** [comment]
now i have tested it.
Firmeware is notthe Problem.

I have Copied the log.

MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x60a423fffe07b655"},"type":"pairing"}'
Error 2023-02-26 10:56:21Publish 'set' 'state' to '0x60a423fffe07b655' failed: 'Error: Command 0x60a423fffe07b655/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 31243 - 1 - 10 - 6 - 11 after 10000ms)'
Info 2023-02-26 10:56:22MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to '0x60a423fffe07b655' failed: 'Error: Command 0x60a423fffe07b655/1 genOnOff.off({}, {\"sendWhen\":\"immediate\",\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (Timeout - 31243 - 1 - 10 - 6 - 11 after 10000ms)'","meta":{"friendly_name":"0x60a423fffe07b655"},"type":"zigbee_publish_error"}'
Error 2023-02-26 10:56:31Failed to configure '0x60a423fffe07b655', attempt 1 (Error: Bind 0x60a423fffe07b655/11 genOnOff from '0x00124b001cdcef21/1' failed (AREQ - ZDO - bindRsp after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (node:internal/timers:564:17) at processTimers (node:internal/timers:507:7))
Info 2023-02-26 10:56:32MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure', payload '{"data":{"id":"0x60a423fffe07b655"},"error":"Failed to configure (Bind 0x60a423fffe07b655/11 genOnOff from '0x00124b001cdcef21/1' failed (AREQ - ZDO - bindRsp after 10000ms))","status":"error","transaction":"b7jmy-1"}'


Maybe the Stick ist damaged. I will send it back.

**Comment by ru320** [comment]
Hello, I now have a solution
Flashing on Windows (https://github.com/Koenkk/zigbee2mqtt/issues/16904)

i will still link all the discussions i had around my problem.
1: https://github.com/Koenkk/zigbee2mqtt/issues/16904
2: https://github.com/Koenkk/zigbee2mqtt/issues/16903
3: https://github.com/Koenkk/Z-Stack-firmware/discussions/429
